src/raft.c

```diff
@@ -306,6 +306,27 @@ void RaftExecuteCommandArray(RedisRaftCtx *rr,
                              RaftRedisCommandArray *cmds)
 {
     RedisModuleCallReply *reply;
+    RedisModuleUser *user = NULL;
+
+    if (cmds->acl) {
+        int nokey;
+        user = RedisModule_DictGet(rr->acl_dict, cmds->acl, &nokey);
+        if (nokey) {
+            char user_name[64];
+            uint64_t count = RedisModule_DictSize(rr->acl_dict);
+            count++;
+            snprintf(user_name, 64, "redis_raft%lu", count);
+            user = RedisModule_CreateModuleUser(user_name);
+            RedisModule_Assert(user != NULL);
+
+            size_t acl_len;
+            const char *acl_str = RedisModule_StringPtrLen(cmds->acl, &acl_len);
+            int ret = RedisModule_SetModuleUserACLString(ctx, user, acl_str, NULL);
+            RedisModule_Assert(ret == REDISMODULE_OK);
+
+            RedisModule_DictSet(rr->acl_dict, cmds->acl, user);
```

Comment:
So, dict might grow over time, we might be okay for now(don't know if people play with ACLs a lot) but it'd be nice to have an issue to remember it.

Reply:
https://redislabs.atlassian.net/browse/RR-93